addons/website/static/lib/jstz.min.js

```diff
@@ -0,0 +1,2 @@
+/* jstz.min.js Version: 1.0.6 Build date: 2015-11-04 */
```

Comment:
you should add the non minified version of the lib and let the bundle of odoo do the minification (like all other libs), also make sure to commit the license if any